gotham-features.md

```diff
@@ -65,3 +65,16 @@ gotham serve --open
 
 - The shortcode has a title parameter.
 - The shortcode has start and stop parameters, measured in seconds.
+
+### QR Code Shortcode
+
+There's a new shortcode available called "qrcode".
+You can pass it a URL and it will output a that URL encoded as a QR code to the page.
+This shortcode shares the HTML and features of the `figure` shortcode minus the embedded link.
+This means parameters such as `alt` and `class` can be used.
+Instead of `width` and `height`, this shortcode has a `size` parameter to specify the sizes of the square.
```

Comment:
only an opinion, but "sizes" seems off. I would use length or dimensions instead.

Reply:
or "specify both width and height of the square"

Reply:
It is off. The word isn't that bad but I think it's worse that I used a plural. I'll go with dimensions.